ui/events/ozone/evdev/keyboard_evdev.cc

```diff
@@ -19,22 +19,15 @@
 
 namespace ui {
 
-namespace {
-
-const int kRepeatDelayMs = 500;
-const int kRepeatIntervalMs = 50;
-
-}  // namespace
-
 KeyboardEvdev::KeyboardEvdev(EventModifiers* modifiers,
                              KeyboardLayoutEngine* keyboard_layout_engine,
                              const EventDispatchCallback& callback)
     : callback_(callback),
       modifiers_(modifiers),
       keyboard_layout_engine_(keyboard_layout_engine),
       weak_ptr_factory_(this) {
-  repeat_delay_ = base::TimeDelta::FromMilliseconds(kRepeatDelayMs);
-  repeat_interval_ = base::TimeDelta::FromMilliseconds(kRepeatIntervalMs);
+  auto_repeat_handler_.SetCallback(
+      base::Bind(&KeyboardEvdev::DispatchKey, weak_ptr_factory_.GetWeakPtr()));
```

Comment:
I don't think you need a weak ptr here as long as KeyboardEvdev will outlive the auto repeat handler. The weak ptr factory is used inside the auto repeat handler, because it can die during the post task. Just pass
base::Unretained(this).

Reply:
Done